src/bullets/portfolio/portfolio.py

```diff
@@ -28,8 +29,15 @@ def __init__(self,
         self.data_source = data_source
         self.slippage_percent = slippage_percent
         self.transaction_fees = transaction_fees
+        self.pending_buy_limit_orders = []
+        self.pending_sell_limit_orders = []
+        self.pending_buy_stop_orders = []
+        self.pending_sell_stop_orders = []
 
     def market_order(self, symbol: str, nb_shares: float):
+        return self.__order(symbol, nb_shares, "Market Order")
```

Comment:
we should differentiate buy and sell market orders depending on nb_shares value (positive/negative) by changing order_type

Reply:
I added it. It's a nice touch.